frontend/src/components/floating/PopperWrapper.vue

```diff
@@ -87,7 +87,6 @@ import {
 const referenceProps = ['class']
 
 export default {
-  name: 'VPopperWrapper',
```

Comment:
This case is different since we create a variant of this component [VPopperWrapper](https://github.com/Akryum/floating-vue/blob/main/packages/floating-vue/src/components/PopperWrapper.vue#L61). I would keep the original name `VPopperWrapper`.

Reply:
✅